engines/dfc_provider/app/serializers/dfc_provider/offer_serializer.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# Serializer used to render the DFC Offer from an OFN Product
+# into JSON-LD format based on DFC ontology
+module DfcProvider
+  class OfferSerializer < ActiveModel::Serializer
+    attribute :id, key: '@id'
+    attribute :type, key: '@type'
+    attribute :offeres_to, key: 'dfc:offeres_to'
```

Comment:
Maybe this should be "offers"?

Reply:
Right, https://github.com/datafoodconsortium/ontology/blob/master/machine%20readable%20files/business%20model/DFC_BusinessOntology.owl#L1674, perdon my french :)

Reply:
Ok, the issue was on the DFC doc, https://datafoodconsortium.gitbook.io/dfc-standard-documentation/technical-specification/api-examples, will update it too.